conda-store-server/conda_store_server/api.py

```diff
@@ -58,28 +58,154 @@ def update_namespace(
     if metadata_ is not None:
         namespace.metadata_ = metadata_
 
-    if role_mappings is not None:
-        # deletes all the existing role mappings ...
-        for rm in namespace.role_mappings:
-            db.delete(rm)
+    return namespace
 
-        # ... before adding all the new ones
-        mappings_orm = []
-        for entity, roles in role_mappings.items():
-            for role in roles:
-                mapping_orm = orm.NamespaceRoleMapping(
-                    namespace_id=namespace.id,
-                    namespace=namespace,
-                    entity=entity,
-                    role=role,
-                )
-                mappings_orm.append(mapping_orm)
 
-        namespace.role_mappings = mappings_orm
+def get_namespace_roles(
+    db,
+    name: str,
+):
+    """Which namespaces can access namespace 'name'?"""
+    namespace = get_namespace(db, name)
+    if namespace is None:
+        raise ValueError(f"Namespace='{name}' not found")
 
-    db.commit()
```

Comment:
This used to have a `db.commit`, which I removed to allow for bulk updates, without committing after every update. That's also what other methods do.